<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="akalanka-Carbon" name="Command line test" tests="2" failures="1" timestamp="27 Apr 2016 08:30:55 GMT" time="0.044" errors="0">
  <testcase name="testAddEmployee" time="0.014" classname="com.unittest.simpletest.EmployeeServiceImplTest">
    <failure type="java.lang.AssertionError" message="expected [5] but found [4]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [4]
	at com.unittest.simpletest.EmployeeServiceImplTest.testAddEmployee(EmployeeServiceImplTest.java:18)
... Removed 39 stack frames]]>
    </failure>
  </testcase> <!-- testAddEmployee -->
  <testcase name="test_AddEmployee_ForExistingEmployee" time="0.0" classname="com.unittest.simpletest.EmployeeServiceImplTest"/>
</testsuite> <!-- Command line test -->
